givenergy_modbus/model/inverter_threephase.py

```
@@ -0,0 +1,544 @@
+    Status,
+    _battery_max_power,
+)
+from givenergy_modbus.model.register import HR, IR, RegisterGetter
+from givenergy_modbus.model.register import Converter as C
+from givenergy_modbus.model.register import RegisterDefinition as Def
+
+
+def _inverter_fault_code2(val: int, word: int) -> list[str] | None:
+    """Decode a 16-bit fault register for three-phase inverters.
+
+    `word` selects one of 9 fault tables (words 0–8), each covering 16 bits.
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)